uptane-standard.md

```diff
@@ -561,20 +561,22 @@ The Timestamp metadata SHALL contain the following information:
 
 Repository mapping metadata informs a primary ECU about which repositories to trust for images or image paths. Repository mapping metadata MUST be present on all primary ECUs, and MUST contain the following information:
 
-* A list of repository names and one or more URLs at which the named repository can be accessed. At a minimum, this MUST include the Director and Image repositories.
-* A list of mappings of image paths to repositories, each of which contains:
+* A list of repository names, each with one or more URLs at which the repository can be accessed.  The first repository name in this list SHALL be known as the Director repository.
```

Comment:
It's a good question.

There are two things at issue here:
1. Should we identify a single Director repository? (IMO, yes)
2. How should we indicate which repository is the Director repository?

**1 is the important question, IMO**, and I tried to get at the rationale for it in [my comment before I made the edits](https://github.com/uptane/uptane-standard/pull/99#issuecomment-484272174), but I'll explain more of my thinking here after tackling 2.

**On question 2**, the notion of making the Director the first in the list was a simplification we could go back and forth on.  It seems cleaner than adding a boolean for each repository indicating Director-ness, and then adding checks in the procedures that require clients to make sure that this property is maintained ("Make sure that one of the repositories listed in the mapping metadata is flagged as a Director repository" or "Make sure that only one..." etc.).  It seems better to establish that the client will expect that it's the first, than to add a bunch of procedural requirements for the clients allowing one more sneaky way metadata can be invalid.  Ira mentioned that he found it preferable, if that's helpful.  If there are better alternatives, great -- just try to prefer simplicity as long as it doesn't constrain implementations much.  (This didn't feel like it did....) 

**More of my thinking on question 1:**
There are different responsibilities that the design assigned to the Director.  Some of these responsibilities work pretty cleanly if we have multiple repositories performing them (ECU identifier limiting, filtering out updates, etc.), and some of them really don't (sole source of Targets metadata to partial verification ECUs, providing and rotating the Timeserver key etc.).  So my thought was to allow things like ECU identifiers in repositories other than "the" Director (hence [this exchange](https://github.com/uptane/uptane-standard/pull/99#discussion_r277099490)), and to isolate the unique properties so that they always come from a single Director repository, and that we just allow other repositories to perform the other responsibilities if an implementer needs that arrangement.  I think it's harder for the procedures in the standard to try to allow for multiple full-fledged Directors.

For example, with the current added here, you can add a repository that serves as a sort of fleet manager that includes ECU IDs and lists all the updates for all the ECUs it wants to permit to a particular vehicle in its top-level Targets metadata, much like a Director.  But we can only provide one Targets role to a partial verification client, and only one repository really needs to handle Timeserver keys (assuming the Timeserver is implemented) so that suggests that one repository should be special: the Director.

Reply:
Yes, exactly: for these reasons, this PR didn't turn out to be as simple as expected, though it's good to clear these things.